consensus/src/simplex/scheme/bls12381_threshold.rs

```diff
@@ -696,14 +696,6 @@ impl<P: PublicKey, V: Variant + Send + Sync> certificate::Scheme for Scheme<P, V
     fn certificate_codec_config_unbounded() -> <Self::Certificate as Read>::Cfg {}
 }
 
-impl<P: PublicKey, V: Variant + Send + Sync> SeededScheme for Scheme<P, V> {
```

Comment:
We can't switch to just using the macro because we "overload" the message, right?